packages/bootleg-app-contracts/migrations/2_deploy_and_mint.js

```diff
@@ -1,17 +1,34 @@
 const BootlegToken = artifacts.require('BootlegToken');
+const BootlegTraderApp = artifacts.require('BootlegTraderApp');
 
-module.exports = async deployer => {
+module.exports = async (deployer, network, accounts) => {
+  let firstOwner = web3.utils.toHex(0);
+  const metaDataURI = 'https://ipfs.infura.io/ipfs/QmSomeHash';
```

Comment:
Similarly hear, maybe have this be an environment variable?

Reply:
ENV VAR ALL THE THINGS

Reply:
But maybe not this. I actually intend to make this a legit URL before we go out to mainnet.